inst/preview/index.html

```diff
@@ -33,7 +33,7 @@
        <h3>{{{pin_name}}}</h3>
        {{#pin_metadata}}
        <p>
-         {{#date}}<b>Last updated:</b> {{{.}}} &bull;{{/date}}
+         {{#date}}<b>Last updated from R:</b> {{{.}}} &bull;{{/date}}
```

Comment:
The Python package can say:

> Last updated from Python:

Reply:
I like this format for a subtle way to be able to tell what language the data is coming from!